### DISCUSSION TITLE
How to change text font

### DISCUSSION THREAD

**Discussion by ICEBEAR63** [mzmine/mzmine#2040] [Q&A]
![image](https://github.com/user-attachments/assets/2d708972-710b-400e-a7da-a91648353460)
from the first time I downloaded mzmine, all words in this software are written _italic_.... 
I want to know how to change default font of mzmine.

**Comment by ansgarkorf** [comment]
Hi,
can you share what operating system you use?

**Reply by ICEBEAR63** [reply]
it's windows 11! 
Thank you
